app/src/lib/parse-app-url.ts

```diff
@@ -70,28 +107,23 @@ export function parseAppURL(url: string): URLActionType {
     // suffix the remote URL with `.git`, for backwards compatibility
     const url = `${probablyAURL}.git`
 
-    const queryString = parsedURL.query
+    const pr = getQueryStringValue(query, 'pr')
+    const branch = getQueryStringValue(query, 'branch')
+    const filepath = getQueryStringValue(query, 'filepath')
 
-    const pr = queryString.pr
-    const branch = queryString.branch
-    const filepath = queryString.filepath
-
-    if (pr) {
-      // if anything other than a number is used for the PR value, exit
+    if (pr != null) {
```

Comment:
We can use the explicit check for `null` since `getQueryStringValue` returns a value or `null`

Reply:
There's a few places where we do this beyond this, and I'd rather leave it as `!=` here to ensure it also catches `undefined`, like the previous code did. How strongly do you feel about this?

Reply:
Not very.